request/context.go

```diff
@@ -33,30 +33,29 @@ type Key string
 var (
 	IDKey         Key = "requestId"
 	OperationKey  Key = "operation"
-	TimestampKey  Key = "requestTS"
+	OriginatorKey Key = "requestOriginator"
 	TenantKey     Key = "tenant"
+	TimestampKey  Key = "requestTS"
 	URLKey        Key = "requestUrl"
-	OriginatorKey Key = "requestOriginator"
+	UserKey       Key = "userId"
 )
 
 // AllKeys is a slice of all Keys
 var AllKeys = []Key{
 	IDKey,
 	OperationKey,
-	TimestampKey,
+	OriginatorKey,
 	TenantKey,
+	TimestampKey,
 	URLKey,
-	OriginatorKey,
+	UserKey,
 }
 
-// HighCardinalityKeys is a map of high-cardinality keys
-var HighCardinalityKeys = map[Key]bool{
-	IDKey:         true,
-	OperationKey:  false,
-	TimestampKey:  true,
-	TenantKey:     false,
-	URLKey:        true,
-	OriginatorKey: false,
+// LowCardinalityKeys is a map of high-cardinality keys
+var LowCardinalityKeys = map[Key]bool{
+	OperationKey:  true,
```

Comment:
IIRC operation includes path parameters and hence may not be low cardinality

Reply:
IIRC, it is the `operationId` from openAPI and the service+method from grpc. But I'll check. `URLKey` is definitely the path parameters